template.yml

```diff
@@ -39,7 +39,7 @@ Globals:
       Variables:
         defaultPowerValues: !Join [ ",", !Ref PowerValues ]
         minRAM: '128'
-        minCost: '0.000000208'
+        minCosts: 'ap-east-1=0.0000002865,af-south-1=0.0000002763,me-south-1=0.0000002583,default=0.000000208'
```

Comment:
Why not a JSON instead? The implementation of `generateBasePriceMap` would be much simpler and elegant.

Reply:
Would you like the json as a env or as a seperate file that is used as:
`const prices = require('./prices.json');`?

Reply:
I was thinking of a JSON string in the environment variable.

Reply:
Cool. Made changes to address this.
Renamed variable to `baseCosts`.